seatunnel-translation/seatunnel-translation-spark/seatunnel-translation-spark-common/src/main/java/org/apache/seatunnel/translation/spark/common/utils/TypeConverterUtils.java

```diff
@@ -85,8 +86,6 @@ public static DataType convert(SeaTunnelDataType<?> dataType) {
                 return DataTypes.BinaryType;
             case DATE:
                 return DataTypes.DateType;
-            case TIME:
-                // TODO: how reconvert?
```

Comment:
I think either modify it to not support first, or don't change it.

Reply:
Or else,we don't suport time Type and throw Exception now. What do you think.